frontend/src/app/models/users/create-user.model.ts

```diff
@@ -2,4 +2,5 @@ export interface CreateUser {
     readonly email: string;
     readonly username: string;
     readonly password: string;
+    readonly inviteCode?: string;
```

Comment:
Nullable so it's safe to remove later

yes I know MongoDB is schemaless _old habits die hard okay_